collisions/func_events_involved.sql

```diff
@@ -3,21 +3,21 @@
 -- DROP FUNCTION IF EXISTS collisions_replicator.refresh_mat_views();
 
 CREATE OR REPLACE FUNCTION collisions_replicator.refresh_mat_views(
-	)
-    RETURNS void
-    LANGUAGE 'sql'
-    COST 100
-    VOLATILE SECURITY DEFINER PARALLEL UNSAFE
+)
+RETURNS void
+LANGUAGE 'sql'
+COST 100
+VOLATILE SECURITY DEFINER PARALLEL UNSAFE
 AS $BODY$
     REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.events WITH DATA;
 	REFRESH MATERIALIZED VIEW CONCURRENTLY collisions_replicator.involved WITH DATA;
 $BODY$;
 
 ALTER FUNCTION collisions_replicator.refresh_mat_views()
-    OWNER TO scannon;
+OWNER TO scannon;
```

Comment:
there might a merge conflict coming, these have changed

Reply:
If the changes are at all significant, we can always scrap this PR and I can just run the fluffer again from master's head. It doesn't take long and would probably be easier than making any manual edits.

Reply:
yea maybe that will be easier.

Reply:
Just let me know when you feel like reviewing and I can do it right then :-)

Reply:
I can review it next week if that works for you!